app/scripts/modules/core/src/pipeline/filter/executionFilter.service.ts

```diff
@@ -137,10 +137,8 @@ export class ExecutionFilterService {
     const searchText = [execution.name];
     searchText.push(execution.id);
     searchText.push(this.getValuesAsString(execution.appConfig));
```

Comment:
@anotherchrisberry do you happen to remember whether `appConfig` is still a valid thing to search on? If not I'll rip it out too.

Reply:
might as well remove it

Reply:
5214a5a